model/location.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+import "go.mongodb.org/mongo-driver/bson/primitive"
+
+type Location struct {
+	ID           primitive.ObjectID `json:"_id,omitempty" bson:"_id,omitempty"`
```

Comment:
I don't think ID should be omitted if it's empty.
```suggestion
	ID           primitive.ObjectID `json:"_id" bson:"_id"`
```